spec/fixtures/_nodes/5x_and_above.json

```diff
@@ -0,0 +1,62 @@
+{
+  "_nodes" : {
+    "total" : 3,
+    "successful" : 3,
+    "failed" : 0
+  },
+  "cluster_name" : "dev",
+  "nodes" : {
+    "Ur_68iBvTlm7Xr1HgSsh6w" : {
+      "name" : "dev-es-master01",
+      "transport_address" : "http://localhost:9200",
+      "host" : "localhost",
+      "ip" : "127.0.0.1",
+      "version" : "5.5.1",
+      "build_hash" : "19c13d0",
+      "roles" : [
+        "master"
```

Comment:
Can we add an additional test case here for a node with `['master', 'data']`? That tests sniffing a default elasticsearch node with no master only stuff.

Reply:
Updated spec to include testing master-only vs master + data node and also fixed the travis config for ES 6.2